src/components/editgrid/EditGrid.js

```diff
@@ -1143,6 +1143,7 @@ export default class EditGridComponent extends NestedArrayComponent {
           const errorContainer = rowContainer.querySelector('.editgrid-row-error');
 
           if (!rowValid && errorContainer && (!this.component.rowDrafts || this.shouldValidateDraft(editRow))) {
+            this.addClass(errorContainer,  'help-block' );
```

Comment:
What if the error is cleared?  Where are we removing the class?

Reply:
This error only shows when you submit the form and there are not saved rows. and right now it doesn't clear the message in prod until you save the row. @travist